### PR TITLE
Python3 migration

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Scipion atlas plugin
 
 This **scipion** plugin will import atlas information from acquisition software
 and will provide some analysis tools to map image processing data
-back to tits grid location.
```
Comment (pconesa) [inline README.rst:7]: :facepalm:

File: atlas/protocols.py [hunk 2]
```diff
@@ -82,46 +95,44 @@ def generateAtlasStep(self, movieDict):
...
+            return parser.getAtlasLocation(movie)
         except Exception as e:
-            print ("EPU parser can't add atlas location for %s. Error: %s" % (movie.getMicName(), e))
+            print("EPU parser can't add atlas location for %s. Error: %s" % (movie.getMicName(), e))
+
+    def _createSetOfAtlasLocation(self, suffix=''):
 
-        setAtlasToMovie(movie, atlasLoc)
+        # Intentionally accessing a private createSet method
+        return self._EMProtocol__createSet(SetOfAtlasLocations,
```
Comment (pconesa) [inline atlas/protocols.py]: In scipion3, Sets have a "factory" classmethod:

SetOfAtlasLocations.create(same params)

Should work.

**Review by pconesa — COMMENTED** [review]
Thanks @azazellochg it looks ok. I made just one comment about creating the Set.

All the collage stuff, I think is trashable. It was my attempt to make our own zoomable gui (google maps like) but failed.

**Comment by azazellochg** [comment]
Thanks, @pconesa . Do you still plan to develop/use the plugin though?

**Pull request merged**